assets/js/modules/analytics/util/report-validation.js

```diff
@@ -34,7 +34,9 @@ import { isValidStringsOrObjects } from '../../../util/report-validation';
 export function isValidMetrics( metrics ) {
 	return isValidStringsOrObjects( metrics, ( metric ) => {
 		const validExpression = metric.hasOwnProperty( 'expression' ) && typeof metric.expression === 'string';
-		const validAlias = metric.hasOwnProperty( 'alias' ) && typeof metric.alias === 'string';
+
+		// 'alias' is optional; if provided, it must be a string.
+		const validAlias = ! metric.hasOwnProperty( 'alias' ) || typeof metric.alias === 'string';
```

Comment:
👍  although the doc block should be updated to reflect this as it currently contradicts this:
> Objects must have "expression" and "alias" properties in order to be considered as valid.